test/integration/utils_test.go

```diff
@@ -179,3 +220,15 @@ func setJobCompletedStatus(job *batchv1.Job) {
 		Status: corev1.ConditionTrue,
 	})
 }
+
+func logPods(t *testing.T) {
```

Comment:
this method seems useful but is unused. Maybe add a code comment why it exists and should not be deleted? Linters may find this, too.

Reply:
keeping as fmt.Print b/c its hard to read logged yaml